src/components/error/OfflineHandler.tsx

```
@@ -0,0 +1,238 @@
+    } catch (error) {
+      console.error('Failed to save data locally:', error);
+    }
+  };
+
+  return (
+    <div className="relative">
+      {/* Offline Warning Banner */}
+      {!isOnline && (
+        <Alert className="mb-4 border-amber-200 bg-amber-50">
+          <WifiOff className="h-4 w-4" />
+          <AlertTitle className="text-amber-900">You're offline</AlertTitle>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)